examples/README.md

```diff
@@ -235,12 +235,12 @@ spec:
           parameters:
           - name: message
             value: "hello2a"
-      - name: hello2b           # single dash => run in parallel with previous step
-        template: whalesay
-        arguments:
-          parameters:
-          - name: message
-            value: "hello2b"
+    - name: hello2b           # single dash => run in parallel with previous step
```

Comment:
this shows better alignment to help indicate that using the object here instead of a nested yaml array means parallelization, and a top level step